chrome/content/zotero/fileInterface.js

```diff
@@ -129,7 +131,8 @@ var Zotero_File_Interface = new function() {
 	 */
 	function exportFile() {
```

Comment:
nit: we should rename this to something like `exportLibrary`, since this name is confusing and inconsistent with functions below. As you're renaming, you can also just use the `this.exportLibrary = function() {...` convention, since that is what Zotero is slowly transitioning to (as opposed to declaring the functions at the top of the object as is done here)